src/core/utils/helpers/general.ts

```diff
@@ -133,3 +133,22 @@ export const sortByLoanDate = (list: LoanV2[]) => {
       new Date(objB.loanDetails.loanDate).getTime()
   );
 };
+
+// If modalids are longer than 0, a modal is open.
+// If a modal is open, the list should not be displayed.
+export const IsAModalDisplayed = (modalIds: string[]) => {
+  return modalIds.length > 0;
+};
+
+export const getRenewableMaterials = (list: LoanV2[]) => {
+  let listOfLoansToCheck: number[] = [];
+
+  listOfLoansToCheck = list
```

Comment:
Is there reason to use the `let listOfLoansToCheck`?
Couldn't we just return right away?:
```
return list.filter(({ isRenewable }) => isRenewable)
    .map(({ loanDetails }) => parseInt(loanDetails.recordId, 10));